src/scp/SCP.cpp

```diff
@@ -120,28 +120,93 @@ SCP::getJsonInfo(size_t limit, bool fullKeys)
     return ret;
 }
 
+SCP::QuorumInfoNodeState
+SCP::getState(NodeID const& node, uint64 slotIndex, int N = 2)
```

Comment:
`N` does not follow our coding standard for variable names

Reply:
Removed.